src/main/java/core/basesyntax/GetDraw.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface GetDraw {

```

Comment:
Interface names should be nouns, not verbs. Consider renaming this interface to something more descriptive, like `Drawable`. This will make your code easier to understand.